untitled-sdl-tank-game/untitled-sdl-tank-game.vcxproj

```diff
@@ -23,7 +23,7 @@
     <ProjectGuid>{c9010400-c373-47b2-9341-8a33c0ba718f}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>untitled_sdl_tank_game</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>

```

Comment:
we will have to take common version of win sdk some time. We will have to switch it manually in Visual. But it is ok for now.